akka-docs/src/main/paradox/general/configuration.md

```diff
@@ -57,7 +57,12 @@ one library in a `reference.conf` of another library.
 Akka requires a license key for use in production. Free keys can be obtained at [https://akka.io/key](https://akka.io/key).
 Add the key to the configuration property `akka.license-key`.
 
-For local development, Akka can be used without a key, but be aware of that the `ActorSystem` will terminate after a while when a key isn't configured.
+For local development, Akka can be used without a key, but be aware of that the `ActorSystem` will terminate after
+a while when a key isn't configured.
+
+If the license key has expired when the `ActorSystem` is started the system will terminate after around 15 minutes.
```

Comment:
I think Tyler did not want to expose a specific time in the documentation, but could be mis-remembering.